src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -538,6 +543,16 @@ theorem toInt_eq_toNat_bmod (x : BitVec n) : x.toInt = Int.bmod x.toNat (2^n) :=
     rw [Int.bmod_neg] <;> simp only [←Int.ofNat_emod, toNat_mod_cancel]
     omega
 
+theorem toInt_le_of_msb_true {x : BitVec w} (h : x.msb = true) : x.toInt ≤ 0 := by
```

Comment:
Is this the strongest statement we can get? I would've expected a `< 0`?

Reply:
Indeed, its `x.toInt < 0`. Fixed.